src/SqlPersistence.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -1,174 +1,207 @@
-﻿namespace NServiceBus.PersistenceTesting
+﻿namespace NServiceBus.PersistenceTesting;
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Newtonsoft.Json;
+using NServiceBus.Extensibility;
+using NServiceBus.Outbox;
+using NServiceBus.Persistence;
+using NServiceBus.Persistence.Sql.ScriptBuilder;
+using NServiceBus.Sagas;
+using NServiceBus.Transport;
+using NUnit.Framework;
+
+public partial class PersistenceTestsConfiguration
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using System.Transactions;
-    using Newtonsoft.Json;
-    using NServiceBus.Extensibility;
-    using NServiceBus.Outbox;
-    using NServiceBus.Persistence;
-    using NServiceBus.Persistence.Sql.ScriptBuilder;
-    using NServiceBus.Sagas;
-    using NServiceBus.Transport;
-    using NUnit.Framework;
-
-    public partial class PersistenceTestsConfiguration
-    {
-        public bool SupportsDtc => OperatingSystem.IsWindows() && ((SqlTestVariant)Variant.Values[0]).SupportsDtc;
-
-        public bool SupportsOutbox => true;
+    public bool SupportsDtc => OperatingSystem.IsWindows() && ((SqlTestVariant)Variant.Values[0]).SupportsDtc;
 
-        public bool SupportsFinders => false;
+    public bool SupportsOutbox => true;
 
-        public bool SupportsPessimisticConcurrency { get; set; }
+    public bool SupportsFinders => false;
 
-        public ISagaIdGenerator SagaIdGenerator { get; private set; }
+    public bool SupportsPessimisticConcurrency => true;
```

Comment:
This is always true since sagas always run in this mode since 4.1 https://docs.particular.net/persistence/sql/saga-concurrency#default-behavior-updating-or-deleting-saga-data